### DISCUSSION TITLE
Welcome to stackexchange Discussions!

### DISCUSSION THREAD

**Discussion by Swaagie** [Swaagie/stackexchange#44] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Swaagie** [comment]
@Trott thank you for actively continuing the development of this package! I must admit that I've been neglecting my open source responsibilities over the last few years due to various reasons. That said, is there anything I can do to assist with development or make sure you get the most from your contributions?

In general I'd like this to be a place were we can async chat a bit about requirements or preferred directions.

**Reply by Trott** [reply]
I'd like to install semantic-release and add a GitHub Action so that the changelog is generated automatically and the release is published automatically when a bugfix, feature, or breaking change is introduced. But that requires an npm token to be added as a secret to the repository. I don't have sufficient access to the repository to add a secret. And I don't know your comfort level with adding a publishing token yourself.

This is a nice-to-have given the infrequent publishing of new versions. It's not a big pain point.

**Reply by Trott** [reply]
Sample semantic-release GitHub Action workflow: https://github.com/Trott/slug/blob/23e6642805be508cfa4318d89c3c1c863bae4bb2/.github/workflows/release.yml

Relevant configuration: https://github.com/Trott/slug/blob/23e6642805be508cfa4318d89c3c1c863bae4bb2/package.json#L46-L68

**Reply by Swaagie** [reply]
I like the idea of doing semantic releases. Added both an `NPM_TOKEN` and `GH_TOKEN` to the repo.
